qunicorn_core/api/deployment_api/deployment_view.py

```diff
@@ -65,7 +67,25 @@ def get(self, deployment_id: int, jwt_subject: Optional[str]):
     def delete(self, deployment_id: int, jwt_subject: Optional[str]):
         """Delete single deployment by ID."""
         logging.info("Request: delete deployment by id")
-        return deployment_service.delete_deployment(deployment_id, user_id=jwt_subject)
+        try:
+            return deployment_service.delete_deployment(deployment_id, user_id=jwt_subject)
+        except QunicornError as ve:
```

Comment:
Das braucht man doch dann auch nichtmehr, wenn man den Status Code setzt oder ?

Reply:
da wird ja extra ein anderer status code (422) behandelt weil das eindeutig kein "Not found" ist ... und wenns nicht der 422 ist wird es normal mit dem 404 behandelt